frontend/ui/elements/reader_menu_order.lua

```diff
@@ -38,12 +38,13 @@ local order = {
         "bookmarks_settings",
     },
     typeset = {
+        "reset_view_settings",
```

Comment:
Can be renamed `reset_document_settings` as it is no longer just "view" settings.